### PR TITLE
Add quantity inverse velocity

### PR DESCRIPTION
**Pull request by groscoe2**
I believe I've changed everything necessary to add inverse velocity (base units are seconds per meter). Please let me know if I missed anything required to merge! I've split this off from the power rate PR to make the changes clearer. I can also recombine them if you'd prefer!

### CODE REVIEW

**Comment by iliekturtles** [comment]
Thanks for the PR. Merged!

**Comment by groscoe2** [comment]
@iliekturtles Certainly, thank you for merging! Out of curiosity, how soon do you expect to have these quantities released on crates.io?

**Comment by iliekturtles** [comment]
I don't have any specific time frame, but I can look to get it done soon for you.

**Comment by iliekturtles** [comment]
@groscoe2 I finally have a PR for [v0.35.0](https://github.com/iliekturtles/uom/pull/423).

**Comment by groscoe2** [comment]
@iliekturtles Thank you for letting me know! I'll be switching back to the main repo as soon as it is released

**Comment by iliekturtles** [comment]
v0.35.0 published!

File: src/si/inverse_velocity.rs [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            "zeptoseconds per meter";
+        @yoctosecond_per_meter: prefix!(yocto); "ym/s", "yoctosecond per meter",
+            "yoctoseconds per meter";
+
+        @hour_per_foot: 1.181_102_362_204_724_3_E4; "h/ft", "hour per foot", "hours per foot";
+        @minute_per_foot: 1.968_503_937_007_873_8_E2; "min/ft", "minute per foot",
+            "minutes per foot";
+        @second_per_foot: 3.280_839_895_013_123; "s/ft", "second per foot", "seconds per foot";
+        @second_per_inch: 3.937_007_874_015_748_E1; "s/in", "second per inch", "seconds per inch";
+        @hour_per_kilometer: 3.6; "h/km", "hour per kilometer", "hours per kilometer";
```
Comment (baarkerlounger) [inline src/si/inverse_velocity.rs:62]: Was there a reason you added `@minute_per_mile` but not `@minute_per_kilometer`? Would there be any objection to adding it and closing https://github.com/iliekturtles/uom/issues/416?

Comment (groscoe2) [reply src/si/inverse_velocity.rs:62]: Only reason was that it wasn't in the base velocity units at the time I put together inverse_velocity. I have no objections to adding it!

**Pull request merged**